.php-cs-fixer.dist.php

```diff
@@ -31,11 +31,6 @@
         'trim_array_spaces' => true,
         'single_blank_line_before_namespace' => true,
         'yoda_style' => false,
-        'global_namespace_import' => [
```

Comment:
why?

Reply:
Could you please elaborate on this?

I don't think adding one tool should alter the behaviour of the other. This PR introduces changes to **175** where, at quick glance, most of them is the effect of changing cs-fixer rules.

Reply:
If you want to keep auto import names, there seems to be a solution for "short classess", too:
```php
$parameters->set(Option::IMPORT_SHORT_CLASSES, false);
```

Reply:
Yes, agree with @marmichalski, pleas do not change cs-fixer rules here, this change is out of scope of this PR